### PR TITLE
Fix Bug 1233620, do not display homepage takeover to Firefox for iOS users

### PR DESCRIPTION
**Pull request by schalkneethling**
@jpetto r?

### CODE REVIEW

**Comment by schalkneethling** [comment]
Just FYI, this one is urgent and should go out to production as soon as we can today. Thanks!

File: media/js/mozorg/home/takeover.js [hunk 1]
```diff
@@ -108,9 +108,10 @@ $(function () {
     // modal is waffled so, first ensure it exists
     if (arrayIndexOfSupported && $takeoverModal.length > 0) {
         try {
-            // only show the takeover modal if the user has not already seen it
-            // during this session.
-            if (sessionStorage.getItem('mozorg.takeover.seen') !== 'true') {
+            // only show the takeover modal if this is not Firefox for iOS and the
+            // user has not already seen it during this session.
+            var takeoverSeen = sessionStorage.getItem('mozorg.takeover.seen');
+            if (isFirefoxiOS() === false && takeoverSeen !== 'true') {
```
Comment (alexgibson) [inline media/js/mozorg/home/takeover.js]: Why not just do `!window.isFirefoxiOS()`?

**Comment by alexgibson** [comment]
LGTM, one small nit but otherwise r+wc :banana:

File: media/js/mozorg/home/takeover.js (hunk 1 above)
Comment (schalkneethling) [inline media/js/mozorg/home/takeover.js]: Ah yes, I did not think to use window as it is implied but, at the same time I contradict myself and try to be explicit by checking against `false` and not `!` :p

**Comment by schalkneethling** [comment]
@alexgibson updated. Thanks!

File: media/js/mozorg/home/takeover.js (hunk 1 above)
Comment (alexgibson) [inline media/js/mozorg/home/takeover.js]: Well, the other conditional is checking the value of a key, not for the existence of the key itself. This is just a simple boolean check :+1:

**Pull request merged**